tools/diagnostics/service_bundle.py

```diff
@@ -149,62 +148,78 @@ def download_log_files(self):
 
     @config.retry
     def create_offers_file(self):
-        scheduler_vip = sdk_hosts.scheduler_vip_host(self.service_name, "api")
-        scheduler = self.scheduler_tasks[0]
+        response = sdk_cmd.service_request("GET", self.service_name, "/v1/debug/offers",
```

Comment:
(nit) wdyt of adding a deprecated annotation here since we are adding a new method now ?!